packages/components/molecules/f-card-with-content/test-utils/component-objects/f-cardWithContent-selectors.js

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line import/prefer-default-export
+export const COMPONENT = '[data-test-id="cardWithContent"]';
```

Comment:
fyi - Accessing this constant fails when calling line _12_ below in file `f-cardWithContent.coponent.js`, so to overcome this issue I recently have embedded this tag into line 12 below and deleted this file completely.

Reply:
Should we raise this with the QE's? I'm guessing that this not working is an issue we may want to solve if this is the pattern they want to keep?